app/uk/gov/hmrc/transitmovementsauditing/services/ObjectStoreService.scala

```diff
@@ -66,4 +75,26 @@ class ObjectStoreServiceImpl @Inject() (client: PlayObjectStoreClient) extends O
         }
     )
 
+  def putFile(id: String, source: Source[ByteString, _])(implicit
```

Comment:
Create some case class and pass that here instead of String

Reply:
Yep - will replace with a Value class.